CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Add `SaleTunnelNotValidated` step into `SaleTunnel`
 - Add WarningIcon component
+- Load React JS bundles from CDN on all pages
```

Comment:
You put this entry in the `2.27.0` release. Do not forgot to move up into `Unrealeased` section

Reply:
Yes, sorry